app/Http/Controllers/Settings/ClientController.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace App\Http\Controllers\Settings;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use Laravel\Passport\Client;
+use Laravel\Passport\ClientRepository;
+
+class ClientController extends Controller
+{
+    /**
+     * Create a new controller instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->middleware(['auth', 'identity']);
+    }
+
+    /**
+     * Create a new api client form
+     *
+     * @return \Illuminate\Contracts\Foundation\Application|\Illuminate\Contracts\View\Factory|\Illuminate\Contracts\View\View
+     *
+     * @throws \Illuminate\Auth\Access\AuthorizationException
+     */
+    public function create()
+    {
+        return view('settings.client.create');
+    }
+
+    public function edit(Client $client)
+    {
+        $client = Client::where('user_id', auth()->user()->id)
```

Comment:
If I call the url with invalid data, it just crashes. Should be an ->authorize() call